packages/terra-clinical-header/src/Header.scss

```diff
@@ -64,8 +64,8 @@
 
   .flex-subheader {
     background-color: #c8cacb;
-    border-bottom-color: initial;
-    border-bottom-style: 0;
+    border-bottom-color: currentColor;
```

Comment:
Curious, if `border-bottom-color` is removed, do we still have the same inherited behavior on the border?

Reply:
The border color does get set to the currentColor if border-bottom-color is removed.